### PR TITLE
media-tv/kodi: add 21.0

### PR DESCRIPTION
**Pull request by parona-source**
* Dropped arm keyword due to new dependency libdisplay-info

Closes: https://bugs.gentoo.org/929014

### CODE REVIEW

**Comment by parona-source** [comment]
Staying as draft due to wayland backend failing to start.

**Comment by parona-source** [comment]
Backend work, but gbm now appears to have issues with getting access to input devices. This wasn't issue before.

File: media-tv/kodi/kodi-21.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,508 @@
...
+	soc? ( system-ffmpeg )
+	udev? ( !libusb )
+	vdpau? ( X !gles !gbm )
+	zeroconf? ( dbus )
+"
+RESTRICT="!test? ( test )"
+
+# dev-libs/libcec[-cubox] bug #818262
+COMMON_DEPEND="
+	>=dev-libs/flatbuffers-1.12.0:=
```
Comment (oniGino) [inline media-tv/kodi/kodi-21.0.ebuild:95]: flatbuffers target version is now  23.3.3

File: media-tv/kodi/kodi-21.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,508 @@
...
+		|| (
+			>=x11-libs/libvdpau-1.1
+			>=x11-drivers/nvidia-drivers-180.51
+		)
+	)
+	wayland? (
+		>=x11-libs/libxkbcommon-0.4.1[wayland]
+	)
+	webserver? (
+		>=net-libs/libmicrohttpd-0.9.55:=[messages(+)]
```
Comment (oniGino) [inline media-tv/kodi/kodi-21.0.ebuild:213]: now targets 0.9.77

**Pull request closed without merging**